requirements.txt

```diff
@@ -76,6 +76,7 @@ requests-oauthlib==1.0.0
 requests==2.21.0
 simplegeneric==0.8.1
 six==1.12.0
+slacker==0.12.0
```

Comment:
https://github.com/polling-bot-4000/polling-bot/blob/70d29e782c15cfd4707991439b49d8aef0374707/polling_bot/brain.py#L4-L14

has served most of my "shove this text into slack" needs, but this looks quite handy for more involved slack API usage (e.g: attachments). Good find.

Reply:
👍